src/grid.js

```diff
@@ -108,6 +108,17 @@ const createGrid = (
     }
   };
 
+  /**
+   * Convert the u,v position to an x,y pixel position
+   * @param   {number}  u  Relative position of the canvas on the x-axis
+   * @param   {number}  v  Relative position of the canvas on the y-axis
+   * @return  {array}  Tuple representing the x,y position
+   */
+  const uvToXy = (u, v) => {
+    const { height } = canvas.getBoundingClientRect();
```

Comment:
I think we can safely move this expression out of `uvToXy` because we'd have to update the entire grid anyway when the height changes otherwise positions would be out of sync.